tests/library/browsercontext-credentials.spec.ts

```diff
@@ -18,8 +18,8 @@
 import { browserTest as base, expect } from '../config/browserTest';
 
 const it = base.extend<{ isChromiumHeadedLike: boolean }>({
-  isChromiumHeadedLike: async ({ browserName, headless }, use) => {
-    const isChromiumHeadedLike = browserName === 'chromium' && (!headless || !!process.env.PLAYWRIGHT_CHROMIUM_USE_HEADLESS_NEW);
+  isChromiumHeadedLike: async ({ browserName, headless, channel }, use) => {
```

Comment:
I don't think this change is needed. We should never run chromium-headless-shell in headed mode.